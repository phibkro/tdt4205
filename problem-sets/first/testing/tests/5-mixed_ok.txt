go   go dx=1 go
go // OK
9: dx=-1 //
10: //
dy=-9     // Not dy=--9
00: dx=-0001 go
94124543535: dx=94124543535 go dy=-111 go // End
