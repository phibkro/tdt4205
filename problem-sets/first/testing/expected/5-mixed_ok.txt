Table filled!
line    1: accepted: go   go dx=1 go
line    2: accepted: go // OK
line    3: accepted: 9: dx=-1 //
line    4: accepted: 10: //
line    5: accepted: dy=-9     // Not dy=--9
line    6: accepted: 00: dx=-0001 go
line    7: accepted: 94124543535: dx=94124543535 go dy=-111 go // End
